src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,586 @@
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>text</dt>
+                    <dd>
+                        The text that has been added to the queue.
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## `utteranceOpts` Options ##
+
+_**Note:** If needed, please read the [Component Configuration Options](ComponentConfigurationOptions.md) document for a full description of infusion component options._
+
+The only option supported by the Text To Speech component is `utteranceOpts`. This option is a javascript object that contains attributes that users can use to define the behaviour of the [SpeechSynthesisUtterance instance](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes) (a part of the web speech API that the Text To Speech component interacts with). These attributes include:
+
+_**Note:** Not all browsers support all these options and some browsers may take different ranges._
+
+### `text` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>text</code> option allows you to set the text that you wish to be spoken. 
+                <strong>Note</strong>: Be careful with this option as it will override any text that was previously passed.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td>
+                ""
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.textToSpeech({
+    utteranceOpts: {
+        text: "Override other texts"
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `lang` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>lang</code> option gives you ability to specify a <a href="http://www.ietf.org/rfc/bcp/bcp47.txt">BCP 47</a> language tag indicating the language of the voice.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td>
+                Default to the language of the HTML document.
+            </td>
+        </tr>
+        <tr>
+            <th>Example</th>
+            <td>
+<pre>
+<code>
+fluid.textToSpeech({
+    utteranceOpts: {
+        lang: "en-US"
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `voiceURI` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>voiceURI</code> option is a string that specifies the speech synthesis voice and the location of the speech synthesis service that the web application wishes to use. Calling the <a href=""#getvoices"">getVoices</a> method returns an array of all available voices, from which you can get the value of the <code>voiceURI</code> attribute.<br />
```

Comment:
There's an extra set of quotes around the url to getVoices which is causing the link to break.